.tx/config

```diff
@@ -1,14 +1,14 @@
 [main]
 host = https://www.transifex.com
```

Comment:
This is still wrong, it turns out....

...but I fixed it with https://github.com/transifex/cli/pull/187

I'll re-run the migration with the fix so that this gets properly updated to `host = https://app.transifex.com`